api/libproject/gcloudcontext.rb

```diff
@@ -0,0 +1,38 @@
+require_relative "../../libproject/utils/common"
+require "json"
+
+class GcloudContextV2
+  attr_reader :account, :project
+
+  def initialize(options_parser)
+    # We use both gcloud and gsutil commands for various tasks. While gcloud can take arguments,
+    # gsutil uses the current gcloud config, so we want to grab and verify the account from there.
+    # We do NOT grab the project from gcloud config since that can easily be set to something
+    # dangerous, like a production project. Instead, we always require that parameter explicitly.
+    common = Common.new
+    options_parser.add_option(
+      "--project [GOOGLE_PROJECT]",
```

Comment:
I think you're going to need to add --creds_file here eventually. Fine if you don't for now.